cluster-provision/okd/4.3/provision.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -x
+
+PARENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )"/../.. && pwd )"
+
+okd_base_hash="sha256:259e776998da3a503a30fdf935b29102443b24ca4ea095c9478c37e994e242bb"
+gocli_image_hash="sha256:a7880757e2d2755c6a784c1b64c64b096769ed3ccfac9d8e535df481731c2144"
+
+gocli="docker run \
+--privileged \
+--net=host \
+--rm -t \
+-v /var/run/docker.sock:/var/run/docker.sock \
+-v ${PARENT_DIR}:${PARENT_DIR} \
+docker.io/kubevirtci/gocli@${gocli_image_hash}"
+
+provisioner_container_id=$(docker ps --filter name=okd-4.3-provision-cluster --format {{.ID}})
+docker kill $provisioner_container_id
+docker container rm $provisioner_container_id
+
+${gocli} provision okd \
+--prefix okd-4.3-provision \
+--dir-scripts ${PARENT_DIR}/okd/scripts \
+--dir-manifests ${PARENT_DIR}/manifests \
+--dir-hacks ${PARENT_DIR}/okd/hacks \
+--workers-memory 8192 \
+--workers-cpu 4 \
+--installer-pull-token-file ${INSTALLER_PULL_SECRET} \
+--installer-repo-tag release-4.3 \
+--installer-release-image registry.svc.ci.openshift.org/origin/release:4.3 \
+"kubevirtci/okd-base@${okd_base_hash}"
+rc=$?
```

Comment:
will `exit $?` work?

Reply:
Yep, I have move the docker kill up, does not make sense to store rc and exit.